content/articles/build-digital-clock-using-javascript/index.md

```diff
@@ -0,0 +1,309 @@
+```
+
+The ```innerHTML``` is a property of the DOM that sets or returns the HTML content of an element. In this case, the content of the div tag with an id of ```Digital_Clock``` is set to show the values of ```currentTime```, ```today```, ```months```, and ```year```. 
+
+The last thing is a function call or invocation. After our function has been declared, given a function body, it will still not work unless it is invoked or called. This is done by placing the function name and brackets immediately after the function's closing curly braces (}).
+
+This is how our final code will look:
+
+```js
+
+setInterval(displayTime, 1000);
+
+function displayTime() {
+    const hoursOfDay = timeNow.getHours();
+    let minutes = timeNow.getMinutes();
+    let seconds = timeNow.getSeconds();
+    let weekDay = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+    let today = weekDay[timeNow.getDay()];
+    let months = timeNow.toLocaleString("default", {
+        month: "long"
+    });
+    let year = timeNow.getFullYear();
+    let period = "AM";
+
+    if (hoursOfDays > 12) {
+        hoursOfDay-= 12;
+        periodOfDay = "PM";
+    }
+
+    if (hoursOfDay == 0) {
+        hoursOfDay = 12;
+        periodOfDay = "AM";
+    }
+
+    hoursOfDay = hoursOfDay < 10 ? "0" + hoursOfDay : hoursOfDay;
+    minutes = minutes < 10 ? "0" + minutes : minutes;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
+
+    let time = hoursOfDay + ":" + minutes + ":" + seconds + period;
+
+    document.getElementById('Clock').innerHTML = time + " " + today + " " + months + " " + year;
+
+}
+displayTime();
+```
+
+Did you see how we intentionally declare all our variables within the function? The goal is to make them in the function's local scope.
+
+#### Third Step
+
+With what we have now, our code should work but we need to add style to it to make it look nice.
+
+We need to create a CSS file and add these styles. I call mine ```index.css```.
+
+```css
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+body {
+     display: flex;
+     align-items: center;
+     justify-content: center;
+     height: 100vh;
+     background: #000;
+     padding: 0;
+     margin: 0;
+     box-sizing: border-box;
+     font-family: 'IBM Plex Sans', sans-serif;
+}
+.glow {
+     font-size: 60px;
+     color: #fff;
+     font-weight: bold;
+     animation: glow 1s ease-in-out infinite alternate;
+     -moz-animation: glow 1s ease-in-out infinite alternate;
+     -webkit-animation: glow 1s ease-in-out infinite alternate;
+}
+```
+
+### Conclusion
+
+In this article, we have learned about function declaration and function calls. We've also learned about the ternary operator and JavaScript date class. Finally, talked about how to get the string value of months in a year using the toLocaleString method.
```

Comment:
I would recommend you add a link the readers can go to to access the final source code. You can put your code on a site like [repl.it](https://replit.com/).